zulip_bots/zulip_bots/bots/calendarbot/calendarbot.py

```diff
@@ -55,8 +55,11 @@ def handle_message(self, message:dict, bot_handler):
                 logging.debug(f"Datetime_input:{datetime_input},length_minutes {length_minutes}")
                 self.create_and_send_meeting(cal, datetime_input, length_minutes)
             case [datetime_input]: 
+                if datetime_input == 'auth':
```

Comment:
@mwen00 In the googlecalendar module, I pulled out the auth functionality from the class to prevent end users from seeing the authenticate with google flow. I know you're working on this part of the code right now - can you account for this?

Reply:
Yes!